integrated_channels/sap_success_factors/exporters/content_metadata.py

```diff
@@ -168,6 +169,10 @@ def transform_schedule(self, content_metadata_item):
         """
         Return the schedule of the content item.
         """
+        if content_metadata_item.get('content_type') == 'course':
+            course_run = get_advertised_or_closest_course_run(content_metadata_item)
+            if course_run:
+                content_metadata_item = course_run
         duration, start, end = get_duration_of_course_or_courserun(content_metadata_item)
```

Comment:
Yep, just had a look at a sample content metadata item in prod and it makes sense.

What if we pass the `content_metadata_item` as is to this method and then use this new utility that I just created `get_advertised_or_closest_course_run` instead of `get_closest_course_run` [here](https://github.com/openedx/edx-enterprise/blob/master/enterprise/utils.py#L1217)?